src/components/AddPlaidBankAccount.tsx

```diff
@@ -187,6 +187,10 @@ function AddPlaidBankAccount({
         onInputChange(plaidAccountID);
     };
 
+    const onError = useCallback((error: ErrorEvent | null) => {
```

Comment:
imo this should be renamed to `handleError`

Reply:
That's a good suggestion. I have renamed it to `handlePlaidError` to be more specific. Thanks!